example_docs/docs/conf.py

```diff
@@ -41,7 +40,7 @@
     'jupyter_sphinx',
     'sphinx_design',
     "nbsphinx",
-    "sphinxcontrib.jquery",
```

Comment:
I figured out now why jQuery wasn't being activated automatically by `qiskit-sphinx-theme` like I hoped for in https://github.com/Qiskit/qiskit_sphinx_theme/pull/277. It's not enough to set `html_theme = qiskit_sphinx_theme`. We need to also add it to `extensions`.

Reply:
If you set `qiskit_sphinx_theme` in `extensions`, then it is redundant to also include `sphinxcontrib.jquery`. That is because we have this code that tells our own extension to also activate jQuery:

https://github.com/Qiskit/qiskit_sphinx_theme/blob/086f163d336de7c827983e62f962a51e5a37b4cb/qiskit_sphinx_theme/__init__.py#L35-L37